deeplearning4j/deeplearning4j-core/src/test/java/org/deeplearning4j/nn/layers/samediff/testlayers/SameDiffConv.java

```diff
@@ -161,14 +160,51 @@ public void applyGlobalConfigToLayer(NeuralNetConfiguration.Builder globalConfig
 
     public static class Builder extends SameDiffLayer.Builder<Builder> {
```

Comment:
There's no real need to modify these "test only" layers, as they aren't part of the public API... though now that it's done maybe there's no harm.